framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1022,6 +1022,37 @@ protected int getCacheSize() {
         }
     }
 
+    /**
+     * Returns an AnnotatedTypeMirror representing the annotated type of {@code elt} as a type
+     * declaration.
+     *
+     * <p>It is not possible to tell whether an element is a use of a type or a declaration, so this
```

Comment:
Technically, an `Element` is always a representation of a declaration, so this description reads a bit strange.